app/controllers/admin/enterprises_controller.rb

```diff
@@ -59,7 +59,7 @@ def bulk_update
     def build_resource_with_address
       enterprise = build_resource_without_address
       enterprise.address = Spree::Address.new
-      enterprise.address.country = Spree::Country.find_by_id(Spree::Config[:default_country_id])
+      enterprise.address.country = Spree::Country.find_by_name(ENV['DEFAULT_COUNTRY'])
```

Comment:
`Spree::Config.default_country_id` is the One Source of Truth for the default country.